<html>
<head>
  <title>Evernote Export</title>
  <basefont face="Segoe UI" size="2" />
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
  <meta name="exporter-version" content="Evernote Windows/308919 (en-US, DDL); Windows/10.0.0 (Win64);"/>
  <style>
    body, td {
      font-family: Segoe UI;
      font-size: 10pt;
    }
  </style>
</head>
<body>
<a name="1876"/>

<div>
<span><div><div style="margin-left: 120px;"><span style="font-size: 18pt; font-family: Arial; font-weight: bold;">RESEARCH PROJECT 1 - NOTEBOOK</span></div><div><span style="font-size: 11pt;"><br/></span></div><div><span style="font-size: 11pt; font-family: Arial;">First download NMR genome assemblies (made up of genomic sequences .fna, annotations .gff, reports .txt etc) from :</span></div><div><font style="font-size: 11pt;"><br/></font></div><ol><li><div><span style="font-size: 11pt; font-family: Arial;">HetGla_1.0 (Kim</span> <span style="font-size: 11pt; font-family: Arial; font-style: italic;">et al</span><span style="font-size: 11pt; font-family: Arial;">, 2011)  </span><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000230445.1" style="font-size: 11pt; font-family: Arial;">https://www.ncbi.nlm.nih.gov/assembly/GCF_000230445.1</a></div></li><li><div><span style="font-size: 11pt; font-family: Arial;">HetGla_female_1.0 (Keane</span> <span style="font-size: 11pt; font-family: Arial; font-style: italic;">et al</span><span style="font-size: 11pt; font-family: Arial;">, 2014)  </span><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000247695.1" style="font-size: 11pt; font-family: Arial;">https://www.ncbi.nlm.nih.gov/assembly/GCF_000247695.1</a></div></li></ol><div><font style="font-size: 11pt;"><br/></font></div><div><span style="font-size: 11pt; font-family: Arial;">Below is a table comparing global statistics for the two genome assemblies of the NMR.</span></div><div><img src="Research Project 1 - Notebook_files/Image.png" type="image/png" data-filename="Image.png"/></div><div><br/></div><div style="box-sizing: border-box; padding: 8px; font-family: Monaco, Menlo, Consolas, &quot;Courier New&quot;, monospace; font-size: 12px; color: rgb(51, 51, 51); border-radius: 4px; background-color: rgb(251, 250, 248); border: 1px solid rgba(0, 0, 0, 0.15);-en-codeblock:true;"><div><b><font style="font-size: 9pt;">CONCLUSION: Both naked mole-rat assemblies (Assembly_1 &amp; Assembly_2) are assembled at the scaffold level, where most scaffolds are unplaced, this means that the objects (scaffolds) do not have a chromosome, linkage-group, or plasmid assignment.</font></b></div><div><b><font style="font-size: 9pt;">Assembly_2 has been described as being a higher quality assembly due to a substantially higher N50 values for scaffold and contigs (Keane et al, 2014).</font></b></div></div><div><br/></div><div style="margin-left: 320px;"><font style="font-size: 11pt;"><br/></font></div><div style="margin-left: 240px;"><span style="font-size: 11pt; font-family: Arial; font-weight: bold; text-decoration: underline;">MAUVE Genome Alignments</span></div><div><font style="font-size: 11pt;"><br/></font></div><ul><li><div><span style="font-size: 11pt; font-family: Arial;">A MAUVE multiple genome alignment (version 2.4.0) between the two published genomes for the naked mole rat was created in a Linux terminal (Bradford University SSH server)</span></div></li></ul><div><span style="font-size: 11pt; font-family: Arial;">            Note: I downloaded 'Putty' (and pscp - &quot;ps copy&quot;) in order to SSH into Dr Andrew Tedder's university server and copy the mauve .xmfa file onto my home PC.</span></div><div><font style="font-size: 11pt;"><br/></font></div><div><span style="font-size: 11pt; font-family: Arial;">Below is the command typed into the terminal to generate the mauve alignment :</span></div><div><br/></div><div style="box-sizing: border-box; padding: 8px; font-family: Monaco, Menlo, Consolas, &quot;Courier New&quot;, monospace; font-size: 12px; color: rgb(51, 51, 51); border-radius: 4px; background-color: rgb(251, 250, 248); border: 1px solid rgba(0, 0, 0, 0.15);-en-codeblock:true;"><div>progressiveMauve --output=H_glaber.xmfa /home/bilalm/H_glaber_genomes/GCA_000230445.1_HetGla_1.0_genomic.fna /home/bilalm/H_glaber_genomes/ GCA_000247695.1_HetGla_female_1.0_genomic.fna</div></div><div> </div><div><span style="font-size: 11pt; font-family: Arial;">The alignment consisting of both NMR assemblies was created using the 'progressiveMauve' algorithm which produced three output files :</span></div><div><font style="font-size: 11pt;"><br/></font></div><div><span style="font-size: 11pt; font-family: Arial;">(1) .XMFA file - containing the Mauve genome alignments</span></div><div><span style="font-size: 11pt; font-family: Arial;">(2) .backbone file - containing records of conserved regions</span></div><div><span style="font-size: 11pt; font-family: Arial;">(3) .bbcols file - containing records of (predicted) alignment positions.</span></div><div><font style="font-size: 11pt;"><br/></font></div><div><span style="font-size: 11pt; font-family: Arial;">Note: It IS possible for the .XMFA file to be graphically visualized using the Mauve-Viewer (version 20150226). This work is still ongoing. </span></div><div><br/></div><div><span style="font-size: 11pt; font-family: Arial;">To view the .XMFA file in the Linux terminal type :</span></div><div><font style="font-size: 11pt;"><br/></font></div><div style="box-sizing: border-box; padding: 8px; font-family: Monaco, Menlo, Consolas, &quot;Courier New&quot;, monospace; font-size: 12px; color: rgb(51, 51, 51); border-radius: 4px; background-color: rgb(251, 250, 248); border: 1px solid rgba(0, 0, 0, 0.15);-en-codeblock:true;"><div>bilalm@Milou:~/H_glaber_genomes/mauve_alignment$ less h_glaber.xmfa</div></div><div><br/></div><div> <img src="Research Project 1 - Notebook_files/Image [1].png" type="image/png" data-filename="Image.png"/></div><div><br/></div><div><span style="font-size: 11pt; font-family: Arial;">Reference:</span></div><div><span style="font-size: 11pt; font-family: Arial;">NNNNNN - Gaps (Ambiguous base calls) </span></div><div><span style="font-size: 11pt; font-family: Arial;">(uppercase) ATCGATCG - confident base calls</span></div><div><span style="font-size: 11pt; font-family: Arial;">(lowercase) atcgatcg - less confident base calls.</span></div><div><font style="font-size: 11pt;"><br/></font></div><div><font style="font-size: 11pt;"><br/></font></div><div><font face="Arial" style="font-size: 11pt;"><br/></font></div><div style="margin-left: 160px;"><span style="font-size: 11pt; font-family: Arial; font-weight: bold; text-decoration: underline;">QUAST - Genome Assembly Assessment Tool</span></div><div><font face="Arial" style="font-size: 11pt;"><br/></font></div><ul><li><div><span style="font-size: 11pt; font-family: Arial;">QUAST is a quality assessment tool for evaluating/comparing genome assemblies.</span></div></li><li><div><span style="font-size: 11pt; font-family: Arial;">QUAST tool in Galaxy (Version 5.0.2) used to generate a number of summary statistics (in numerical/graphical format) in order to gauge an assembly quality for the naked mole-rat genomes. </span></div></li></ul><div><font face="Arial" style="font-size: 11pt;"><br/></font></div><div><span style="font-size: 11pt; font-family: Arial; font-weight: bold;">Step 1</span> <span style="font-size: 11pt; font-family: Arial;">- Download </span><span style="font-size: 11pt; font-family: Arial; line-height: 107%;">both genomes (HetGla_1.0 &amp; HetGla_female_1.0) from the NCBI website under accession numbers ‘GCF_000230445.1’ &amp; ‘GCF_000247695.1’ in FASTA/.fna format. </span></div><div><font face="Arial" style="font-size: 11pt;"><br/></font></div><div><span style="font-size: 11pt; font-family: Arial; font-weight: bold;">Step 2</span> <span style="font-size: 11pt; font-family: Arial;">- A d</span><span style="font-size: 11pt; font-family: Arial;">ecompression step was required as the newly downloaded .fna files were in a compressed .gz format. To compress the genome sequence files use: </span></div><div><br/></div><div style="box-sizing: border-box; padding: 8px; font-family: Monaco, Menlo, Consolas, &quot;Courier New&quot;, monospace; font-size: 12px; color: rgb(51, 51, 51); border-radius: 4px; background-color: rgb(251, 250, 248); border: 1px solid rgba(0, 0, 0, 0.15);-en-codeblock:true;"><div><span style="font-size: 9pt; color: rgb(51, 51, 51); font-family: Monaco;">$ gunzip </span>GCA_000230445.1_HetGla_1.0_genomic.fna.gz</div><div><span style="font-size: 9pt; color: rgb(51, 51, 51); font-family: Monaco;">$ gunzip </span>GCF_000247695.1_HetGla_female_1.0_genomic.fna.gz</div></div><div><br/></div><div><br/></div><div><span style="font-size: 11pt; font-family: Arial;"><span style="font-size: 11pt; font-family: Arial; font-weight: bold;">Step 4</span> - Import the decompressed FASTA files into Galaxy. Wait for upload, then select the </span><span style="font-size: 11pt; font-family: Arial;">Quast Genome assembly Quality (Galaxy Version 5.0.2+galaxy1) tool. </span></div><div><span style="font-size: 11pt;"><br/></span></div><div><span style="font-size: 11pt; font-family: Arial;"><span style="font-size: 11pt; font-family: Arial; font-weight: bold;">Step 5</span> - Wait for QUAST to run. Quast generates (1) tabular report (2) HTML report (3) PDFG report ($) Log .txt file.  </span></div><div><span style="background-color: rgb(255, 250, 165);-evernote-highlight:true;"><br/></span></div><div><span style="background-color: rgb(255, 250, 165); font-size: 11pt; font-family: Arial;-evernote-highlight:true;">Note: Results are not included in this notebook. To see the results of QUAST for NMR genomes go to :  https://github.com/mbilal1995/Research-Project_1</span></div><div><br/></div><div><br/></div><div style="margin-left: 160px;"><font style="font-size: 11pt;"><span style="font-size: 11pt; font-family: Arial; font-weight: bold; text-decoration: underline;">Bioinformatic Pipeline - De Novo Genome Assembly</span></font></div><div style="margin-left: 320px;"><font style="font-size: 11pt;"><br/></font></div><ul><li><div><span style="font-size: 11pt; font-family: Calibri, sans-serif; line-height: 107%;">In the future I intend to build a full</span> <span style="font-size: 11pt; font-family: Calibri, sans-serif; font-style: italic; line-height: 107%;">de novo</span> <span style="font-size: 11pt; font-family: Calibri, sans-serif; line-height: 107%;">genome assembly of naked mole-rat reads based off Illumina sequencing data from the NCBI website.</span></div></li><li><div><span style="font-size: 11pt; font-family: Calibri, sans-serif; line-height: 107%;">The research question will be: “Working with the existing Illumina sequencing data, is It possible to make an improved</span> <span style="font-size: 11pt; font-family: Calibri, sans-serif; font-style: italic; line-height: 107%;">de novo</span> <span style="font-size: 11pt; font-family: Calibri, sans-serif; line-height: 107%;">assembly from either the reads of one or both of the NMR genomes?”</span></div></li></ul><div><br/></div><div><span style="font-size: 11pt; text-indent: 0mm; color: rgb(1, 1, 1); font-family: Calibri;">The bioinformatic pipeline consisting of the major steps I will use to solve the research question are set out in the black box below :</span></div><div><img src="Research Project 1 - Notebook_files/Image [2].png" type="image/png" data-filename="Image.png"/></div><div><br/></div><div>A graphical representative of the bioinformatic pipeline.</div><div><br/></div><div><img src="Research Project 1 - Notebook_files/Image [3].png" type="image/png" data-filename="Image.png" width="1006"/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div><div><br/></div></div><div><br/></div></span>
</div></body></html> 